app/src/main/kotlin/com/hippo/ehviewer/ui/main/EhAsyncThumb.kt

```diff
@@ -57,7 +58,9 @@ fun EhAsyncCropThumb(
     AsyncImage(
         model = requestOf(key),
         contentDescription = null,
-        modifier = modifier.sharedBounds(key = "${key.gid}").clip(ShapeDefaults.Medium),
+        modifier = modifier
+            // .sharedBounds(key = "${key.gid}")
```

Comment:
This comment out is not necessary if we return `this` in sharedBounds in NoopSharedTransitionScope

Reply:
`SharedContentState`'s constructor is internal